test/_init_tests.js

```diff
@@ -402,9 +402,8 @@ function beforeTest() {
   activateChunkingForTesting();
   window.__AMP_MODE = undefined;
   window.context = undefined;
-  window.AMP_CONFIG = {
-    canary: 'testSentinel',
-  };
+  window.AMP_CONFIG = require('../build-system/global-configs/prod-config.json');
+  window.AMP_CONFIG.canary ='testSentinel'
```

Comment:
Nice! For posterity, can you confirm that the initialization worked by printing a debug message for `window.AMP_CONFIG` after these two lines? (Local verification is fine, no need to check in the debug message.)

Reply:
Done! Looked fine to me. Although tbh I have no idea what the `canary = testSentinel` is for

Reply:
I looked at the blame and noticed that `testSentinel` was added 4 years ago in #4878, and is currently not used anywhere in dev or test code. Perhaps you can try removing the line and see what difference it makes to the tests?

Reply:
Another point in favor of removing `testSentinel`: Now that we're looking to ensure that the prod config is in use, I think we should just allow this built-in value of `canary: 0` to be used.

https://github.com/ampproject/amphtml/blob/f2a37e85f60827744c0689027e379169b53d1325/build-system/global-configs/prod-config.json#L8